sonar-db/src/test/java/org/sonar/db/component/ComponentDbTester.java

```diff
@@ -83,8 +83,13 @@ public void insertComponents(ComponentDto... components) {
     db.commit();
   }
 
-  public void indexProjects() {
+  public void indexProjectsAndViews() {
     dbClient.componentIndexDao().indexProjects();
```

Comment:
I didn't know this method was used only for tests purpose. In the context of this PR I will stop using it in the new code